### DISCUSSION TITLE
Official Umbraco Docker Image

### DISCUSSION THREAD

**Discussion by jstarkwv** [umbraco/Umbraco-CMS#10988] [Features and ideas]
Could the Umbraco team create an official docker image on docker hub for Umbraco 9? 

[Docker Hub](https://hub.docker.com/search?q=&type=image)

I think this will help grow the popularity of Umbraco if you had a simple container install that can run on any OS using .NET Core.

**Comment by nul800sebastiaan** [comment]
It's on our todo list, but we lack the internal expertise at the moment. So we need to learn, get familiar with the tooling and find a few people in the company who can be responsible for it.

**Reply by Curious-r** [reply]
> It's on our todo list, but we lack the internal expertise at the moment. So we need to learn, get familiar with the tooling and find a few people in the company who can be responsible for it.

Sorry for necroposting, but I wanted to ask if there is any progress?

**Reply by bergmania** [reply]
Not yet

**Comment by bergmania** [comment]
Another thing we need to verify is how useful an official image is. 
As of Umbraco 11 I dont see it useful, due to the missing headless api.

From Umbraco 12 it could make sense, as Umbraco is useful out-of-box. 

What I think sounds more interesting, is to be able to build to a docker image from the Umbraco.Templates. Then all the customisations can be part of the image.
